arelle/PrototypeInstanceObject.py

```diff
@@ -4,10 +4,11 @@
 from arelle import XmlUtil
 from arelle.ModelValue import QName
 from arelle.ModelObject import ModelObject
+from typing import Any
 Aspect = None
 
 class FactPrototype():      # behaves like a fact for dimensional validity testing
-    def __init__(self, v, aspectValues=None):
+    def __init__(self, v, aspectValues=None) -> Any:
```

Comment:
`__init__` always returns `None`:
```suggestion
    def __init__(self, v, aspectValues=None) -> None:
```